code-reviews/pull-requests.md

```diff
@@ -33,17 +33,31 @@ We should always aim to have pull requests be as small as possible, without losi
 1. They are easier to deploy; this is aligned with the strategy of release fast and release often.
 1. Minimizes possible conflicts and stale PRs, which are difficult to merge and keep in sync with master either because they're very dynamic or contain refactoring.
 
-However, we should avoid having PRs that include code that is without context or loosely coupled.
+However, we should avoid having PRs that include code that is without context or loosely coupled. There is no right size, but keep in mind that a code review is a collaborative process, and you should work with your reviewer to find out a reasonable size. Besides the size, remember that every PR should:
```

Comment:
I agree, I also don't understand how this would work.  In general you won't know who your reviewer is on beforehand, and IMHO the PR size should not depend on reviewer. We should always strive to have as small PRs as possible that still add value.

I believe this section could be simplified to that.  If we don't feel we can do small PRs this is usually because PRs take too long to be reviewed (so we believe we need to do larger PRs to make progress) in which case the solution is not larger PRs but a better PR process. 

This is a good resource for the PR size topic: https://google.github.io/eng-practices/review/developer/small-cls.html

Reply:
Fair point, This was actually based on the google PR size topic on the section "What is small?". I'll try to simplify that.

Reply:
From the Google size topic, it actually suggest that the PR size is at reviewer judgement:

"There are no hard and fast rules about how large is “too large.” 100 lines is usually a reasonable size for a CL, and 1000 lines is usually too large, but it’s up to the judgment of your reviewer."

But I understand, that, I may only add a reference to the link and simplify this wording.

Reply:
Added the google link as reference, simplifying wording. I added one comment you made Tess, I think that explains pretty good the point: "We should always strive to have as small PRs as possible that still add value.".